src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -121,9 +146,7 @@ public Internship findInternshipById(InternshipId internshipId) {
      * Updates the personIdCounter to avoid duplicate Ids.
      */
     public void addPerson(Person p) {
```

Comment:
Should this method (along with `UniquePersonList::add`) throw a `DuplicatePersonException`?